ArWikiCats/legacy_bots/with_years_bot.py

```
@@ -160,18 +166,20 @@ def _handle_year_at_end(
         or wrap_lab_for_country2(remainder)
         or ""
     )
+    if not remainder_label:
+        return ""
     if "–present" in formatted_year_label:
```

Ruff (0.14.13)

String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)